addons/aws/external-dns/variables.tf

```diff
@@ -52,21 +55,21 @@ locals {
   default_tags = merge(local.tags, var.tags)
 }
 
-variable "external_dns_annotation_filter" {
-  default = ""
-}
+# variable "external_dns_annotation_filter" {
```

Comment:
is this cleanup activity?  should be another PR for cleanup or if its required for changes should be deleted.